schedule/yast/maintenance/yast_mru_install_minimal_with_addons_restapi_pcm.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
the name should not reflect the technology, "restapi" and "pcm" is not really clear what it means,
Do you know why the three schedules for all the architectures where is running this test suite are not the same at the moment?

Reply:
The only arch that this failure has been seen is s390x as far as I remember. I guess we could use select_patterns for all archs, but then again, as you have mentioned the module is far from ideal.

Reply:
_restapi was there in the original schedule, any recommendation for naming? Honestly, I don't even try anymore because I know you will not like it :grin: